app/ide-desktop/lib/client/src/url-associations.ts

```diff
@@ -0,0 +1,147 @@
+/** @file URL associations for the IDE. */
+
+import * as electron from 'electron'
+import electronIsDev from 'electron-is-dev'
+
+import * as common from 'enso-common'
+import * as contentConfig from 'enso-content-config'
+
+const logger = contentConfig.logger
+
+export function registerAssociations() {
```

Comment:
new file is missing sections - not sure it's required here, but afaict our code style requires sections in every file

Reply:
Yes, its required, thanks for catching it!

Reply:
Also, this is an exported function, it should have docs. Why linter did not catch it?

Reply:
i don't think the linter is currently set to error on missing docs, maybe i should look for an eslint plugin to try and enforce htis?